### PR TITLE
implementation logic of copyAndPaste task

### PR DESCRIPTION
**Pull request by sergtm1**

### CODE REVIEW

File: src/main/java/com/sergtm/application/copyAndPaste/RegularBankAccount.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    private String number;
+    private double balance;
+    private Integer amountOfCostlyOperations;
+
+    public String toString(){
+        StringBuilder builder = new StringBuilder();
+        builder.append("Number: ").append(number).append(", Balance: ").append(balance);
+        return builder.toString();
+    }
+    public void displayRegularBA(RegularBankAccount regularBankAccount){
```
Comment (sergtm1) [inline src/main/java/com/sergtm/application/copyAndPaste/RegularBankAccount.java]: Move this function to Main class

File: src/main/java/com/sergtm/application/copyAndPaste/RegularBankAccount.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package com.sergtm.application.copyAndPaste;
+
+public class RegularBankAccount {
+
+    private String number;
+    private double balance;
+    private Integer amountOfCostlyOperations;
+
+    public String toString(){
```
Comment (sergtm1) [inline src/main/java/com/sergtm/application/copyAndPaste/RegularBankAccount.java]: Add @Overide anotation

Comment (sergtm1) [inline src/main/java/com/sergtm/application/copyAndPaste/RegularBankAccount.java]: Move toString to bottom of class

File: src/main/java/com/sergtm/application/copyAndPaste/SavingsBankAccount.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
+package com.sergtm.application.copyAndPaste;
+
+public class SavingsBankAccount {
+    private String number;
```
Comment (sergtm1) [inline src/main/java/com/sergtm/application/copyAndPaste/SavingsBankAccount.java]: Rename it to accountNumber

File: src/main/java/com/sergtm/application/copyAndPaste/RegularBankAccount.java (hunk 1 above)
Comment (aleksey-timofeev) [inline src/main/java/com/sergtm/application/copyAndPaste/RegularBankAccount.java]: done

File: src/main/java/com/sergtm/application/copyAndPaste/Main.java [hunk 4]
```diff
@@ -57,6 +57,13 @@ public void displaySavingBAList( ArrayList<SavingsBankAccount> listOfSavingBankA
             System.out.println(s);
         }
     }
+    public void  displaySavingsBA(SavingsBankAccount savingsBankAccount){
```
Comment (aleksey-timofeev) [inline src/main/java/com/sergtm/application/copyAndPaste/Main.java]: Change instance methods to static ones

**Pull request merged**